di_website/templates/includes/heroes/hero-event.html

```diff
@@ -0,0 +1,38 @@
+{% load wagtailimages_tags wagtailcore_tags responsive %}
```

Comment:
How does this `hero` differ from the existing ones? However it does, it can still extend the `hero.html` to avoid repetition.

Reply:
If this isn't being used anymore, please remove it.

Reply:
It now extends hero.html and is being used in event_page.html